src/index.js

```diff
@@ -1,12 +1,23 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
-import App from './App';
+import Calculator from './components/calculator';
 import reportWebVitals from './reportWebVitals';
 
+class CalculatorApp extends React.PureComponent {
+  render() {
+    return (
+      <div className="container">
+        <h1>The Calculator Web App</h1>
+        <Calculator />
+      </div>
+    );
+  }
+}
+
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+    <CalculatorApp />
   </React.StrictMode>,
   document.getElementById('root'),
 );
```

Comment:
- [ ] Could you please split out App.js and index.js. 

App.js should be inside the src/App.js

`class CalculatorApp extends React.PureComponent {
  render() {
    return (
      <div className="container">
        <h1>The Calculator Web App</h1>
        <Calculator />
      </div>
    );
  }
}`

and index.js file should be like this.

`ReactDOM.render(
  <React.StrictMode>
    <App />
  </React.StrictMode>,
  document.getElementById('root'),`

Reply:
Requirement satisfied.